src/mame/skeleton/monon_color.cpp

```diff
@@ -548,6 +557,44 @@ void monon_color_state::do_palette(int amount, int pal_to_use)
 
 void monon_color_state::write_to_video_device(uint8_t data)
 {
+	static const char* names[] =
```

Comment:
You’ve made an array of non-const pointers here.

Reply:
This idiom is used all over the codebase, particularly my devices and drivers. Please refer to an example of it being done correctly.

Reply:
You two `const` qualifiers – one for the pointers, and one for the things they point to.  This kind of error is harder to make if you always put the `const` qualifier to the right of the type, as then it’s obvious it applies to the thing immediately to its left.

```
char *a; // non-const pointer to const char
char const *b; // non-const pointer to const char
char *const c; // const pointer to non-const char
char const *const d; const pointer to const char
```

Reply:
Edited comment to correct comments since I can’t code while multi-tasking.

Reply:
I explained this before in comments on 556ba59efc26b8a198094af2f56fdc8fb5c6e6bb.  I also fixed a pile of them in 2993e850bbe44f8e7992877bd1e5a767f762b1ac, and at other times when checking for non-const static variables.

Reply:
That's reasonable. It seems that there was a bulk cleanup pass done at some point to const-correct most string arrays, as it wasn't always like this in the codebase. I haven't done a comprehensive search for all possible combinations of spacing between the pointer operator and "const", so there are probably other cases, but would a pull request be accepted to fix up the following files that do it wrong?

```src\devices\bus\coco\cococart.cpp
src\devices\bus\nes\nes_ines.hxx
src\devices\machine\i7220.cpp
src\devices\machine\mc6843.cpp
src\devices\machine\wd33c9x.cpp
src\devices\sound\megd.cpp
src\mame\konami\3dom2_te.cpp
src\mame\nintendo\n64_v.cpp
src\mame\yamaha\ymmu100.cpp
src\osd\modules\render\drawogl.cpp
```

Reply:
> would a pull request be accepted to fix up the following files that do it wrong

Of course it would.  I appreciate any help in fixing common errors.

Reply:
resolved, I think

Reply:
Yep,that does make them const.